pacer.js

```diff
@@ -113,11 +113,17 @@ let PACER = {
     for (let url of urls) {
       let hostname = getHostname(url);
       // JS is trash. It lacks a way of getting the TLD, so we use endsWith.
+      //
+      // xxx aren't we only invoked when we match the "matches" and
+      // "include_globs" in manifest.json, which limits us to
+      // *://*.uscourts.gov/* ?
       if (hostname.endsWith('uscourts.gov')) {
-        let match = url.match(/\?(\d+)$/);
-        if (match) {
+        let match;
+        if (match = url.match(/\?(\d+)$/)) {
```

Comment:
I just noticed, but I think this still needs a tweak to remove the `$` on the end (like you did above). Otherwise, a URL like this won't work:

https://ecf.almd.uscourts.gov/cgi-bin/DktRpt.pl?38115&

I just stumbled on this because of this ticket. Looks like we can fix it too while we're in here: https://github.com/freelawproject/recap/issues/168

Reply:
Sure. I was trying to do the minimal change necessary to support the feature I was working on, but we can absolutely relax this, done.